theolive/platform/security/token-based-security.mdx

```diff
@@ -56,13 +56,16 @@ First thing to do is to create a header provider function. In this function you
 
 ```typescript Creating header provider
 const interceptor = (request) => {
-  const headers = {
-    ...request.headers,
-    Authorization: 'Bearer <token-generated-by-you>',
-  };
-  request.redirect({
-    headers,
-  });
+  // Only add 'Authorization' header for THEOlive request to avoid CORS issues
+  if (new URL(request.url).hostname.endsWith("theo.live")) {
+    const headers = {
+      ...request.headers,
+      Authorization: `Bearer ${params.token}`,
```

Comment:
The original code snippet made it clear that the token needed to be generated by some custom code. I think that's a bit less obvious from just `params.token`.